independent-publisher-connectors/Influenza and Covid-19/apiDefination.swagger.json

```diff
@@ -0,0 +1,9216 @@
+                      "9/29/22": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "9/29/22"
+                      },
+                      "9/30/22": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "9/30/22"
+                      },
+                      "10/1/22": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "10/1/22"
+                      },
+                      "10/2/22": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "10/2/22"
+                      },
+                      "10/3/22": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "10/3/22"
+                      },
+                      "10/4/22": {
+                        "type": "integer",
+                        "format": "int32",
+                        "description": "10/4/22"
+                      }
+                    },
+                    "description": "timeline"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get COVID-19 vaccine doses administered for all states",
+        "description": "Get COVID-19 vaccine doses administered for all states that have reported rolling out vaccination. Sourced  from covid.ourworldindata.org",
+        "operationId": "GetCOVID-19vaccinedosesforallstates",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "lastdays",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "name": "fullData",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      }
+    },
+    "/v3/covid-19/vaccine/coverage/states/{state}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "state": {
+                  "type": "string",
+                  "description": "state"
+                },
+                "timeline": {
+                  "type": "object",
+                  "properties": {
+                    "9/5/22": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "9/5/22"
+                    },
+                    "9/6/22": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "9/6/22"
+                    },
+                    "9/7/22": {
+                      "type": "integer",
+ 
```

Comment:
@kelcho-spense All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)

Reply:
I have rectified the error.